os/fs/driver/mtd/smart.c

```diff
@@ -493,73 +493,6 @@ static void smart_free(FAR struct smart_struct_s *dev, FAR void *ptr)
 }
 #endif
 
-/***************************************************************************************
- *   This is called from the fs_clean() API to release the memory acquired by MTD driver
- **************************************************************************************/
-int smart_clean(FAR struct inode *inode)
```

Comment:
We need this function to free when fs unmounted.
I have a plan that use this function with ioctl